src/Storageless/Session/Zend/SessionAdapter.php

```diff
@@ -0,0 +1,105 @@
+<?php
+/*
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+ * "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT
+ * LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR
+ * A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT
+ * OWNER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL,
+ * SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT
+ * LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE,
+ * DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY
+ * THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+ * (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+ * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ *
+ * This software consists of voluntary contributions made by many individuals
+ * and is licensed under the MIT license.
+ */
+
+declare(strict_types=1);
+
+namespace PSR7Sessions\Storageless\Session\Zend;
+
+use Lcobucci\Clock\Clock;
+use Lcobucci\Clock\SystemClock;
+use PSR7Sessions\Storageless\Session\SessionInterface;
+use Zend\Expressive\Session\SessionInterface as ZendSessionInterface;
+
+final class SessionAdapter implements ZendSessionInterface
+{
+    public const SESSION_REGENERATED_NAME = '_regenerated';
+
+    /** @var SessionInterface */
+    private $session;
+
+    /** @var Clock */
+    private $clock;
+
+    public function __construct(SessionInterface $session, ?Clock $clock = null)
+    {
+        $this->session = $session;
+        $this->clock   = $clock ?? new SystemClock();
+    }
+
+    /**
+     * @return array
+     */
+    public function toArray() : array
+    {
+        return (array) $this->session->jsonSerialize();
+    }
+
+    /**
+     * @param mixed $default
+     * @return mixed
```

Comment:
There is no need for the documentation as the parent method is already documenting it and we are adding no value with it here, I'd suggest using `@inheritDoc` if the doc is really relevant here.